lib/Coocook/Model/RecipeImporter.pm

```diff
@@ -46,8 +46,7 @@ sub _build_ingredients { return [ shift->recipe->ingredients_sorted->hri->all ]
 
 sub _build_source_articles {
     return [
-        shift->recipe->ingredients->search_related( 'article', undef, { distinct => 1 } )->hri->all
-    ];
+        shift->recipe->ingredients->search_related( 'article', undef, { distinct => 1 } )->hri->all ];
```

Comment:
Weird, why was this incorrect formatted before? `xt/perltidy.t` should have prevented that:-/

* New version is stable
* current `master` is not
* tested with perltidy-sweet v20200619

Reply:
I also wondered how this could happen, because I never touched those files. But I included these changes, so that the Travis build passes, because it was quite distracting that Travis complained all the time about failing the tests. I think that before merging this PR into `master` we should push these commits seperately and then merge this PR.